data.sql

```diff
@@ -40,3 +40,36 @@ INSERT INTO species (name) VALUES ('Digimon');
 UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
 UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
 
+-- insert data into the vets table
+INSERT INTO vets (name, age, date_of_graduation) 
+VALUES ('William Tatcher', 45, 'Apr 23, 2000'),
+('Maisy Smith', 26, 'Jan 17, 2019'),
+('Stephanie Mendez', 64, 'May 4, 1981'),
+('Jack Harkness', 38, 'Jun 8, 2008');
+
+-- insert data into the specializations table
+INSERT INTO specializations (vets, species) VALUES (1, 1),
+(3, 2),
+(4, 2);
```

Comment:
- [ ] Good job so far 👏  But I think, you need to Insert the following data for specialties Both  Vet Stephanie Mendez is specialized in Digimon and Pokemon, and I can see only one query. Please make sure to add the missing query 👍

Reply:
Hello @codecaiine , Thanks. I will make the changes and request a new review